<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortUserWindow</class>
 <widget class="QDialog" name="SortUserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактировать отображение</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_sort">
     <property name="text">
      <string>Сортировки: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_type_sort">
     <item>
      <property name="text">
       <string>сортировка по номеру</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>сортировка по исходящим</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>сортировка по входящим</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_filter">
     <property name="text">
      <string>Фильтрация: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_type_filter">
       <item>
        <property name="text">
         <string>без фильтрации</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>физические лица с отрицательным балансом</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>выбранный юр. лицо</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_value_filter"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_save">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
